File: core/core-api/src/main/java/io/say/better/global/common/response/JwtTokenResponseDto.java
package io.say.better.global.common.response;

import java.time.LocalDateTime;

import com.fasterxml.jackson.annotation.JsonProperty;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.ToString;

@Getter
@Builder
@ToString
@AllArgsConstructor
public class JwtTokenResponseDto {

	private String accessToken;
	private String refreshToken;
	private LocalDateTime expiredTime;
	@JsonProperty("isExisted")
	private boolean isExisted;

}

File: core/core-api/src/main/kotlin/io/say/better/global/common/response/JwtTokenResponseDto.kt
package io.say.better.global.common.response

import com.fasterxml.jackson.annotation.JsonProperty
import lombok.AllArgsConstructor
import lombok.Builder
import lombok.Getter
import lombok.ToString
import java.time.LocalDateTime

@Getter
@Builder
@ToString
@AllArgsConstructor
class JwtTokenResponseDto {
    private val accessToken: String? = null
    private val refreshToken: String? = null
    private val expiredTime: LocalDateTime? = null

    @JsonProperty("isExisted")
    private val isExisted = false
}
